tox.ini

```diff
@@ -27,7 +27,7 @@ deps =
     oldest: scikit-learn==1.0.1
     oldest: orange3==3.34.0
     oldest: orange-canvas-core==0.1.28
```

Comment:
Tests started to fail since the widget-base needs a newer version of canvas-core than it is set her. Just change it to 0.1.30.

```suggestion
    oldest: orange-canvas-core==0.1.30
```

Reply:
And you can commit the change just with click on Commit suggestion button